terraform-modules/spectrum/templates/cilium.yml

```diff
@@ -45,16 +50,10 @@ hubble:
   ui:
     enabled: true
     rollOutPods: true
+%{ endif }
 
-%{ if l2_enabled }
 l2announcements:
   enabled: true
 
 externalIPs:
   enabled: true
-
-devices:
```

Comment:
removed, devices not required. But if we decide to have it - will be managed at fcli manifests